src/k8s-engine/lima.ts

```diff
@@ -356,6 +357,12 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     return stdout;
   }
 
+  limaSpawn(args: string[]) : ChildProcess {
```

Comment:
This is basically the same as `ssh`, except returning the child process, right? (That should be okay, just checking to make sure I'm not missing anything.)

Reply:
Yes, it is. Did it this way to avoid duplicating child process output (yet again).